src/api/app/controllers/webui/webui_controller.rb

```diff
@@ -368,9 +366,9 @@ def set_influxdb_data
   end
 
   def set_influxdb_additional_tags
-    anonymous = User.current_login == User::NOBODY_LOGIN
+    anonymous = !User.session
     tags = {
-      beta: !anonymous && User.current.in_beta?,
+      beta: !anonymous && User.possibly_nobody.in_beta?,
```

Comment:
A really similar change happened in another file in this PR, but it was done differently:

From:

```
anonymous = User.current_login == User::NOBODY_LOGIN

InfluxDB::Rails.current.tags = {
  beta: !anonymous && User.current.in_beta?,
  anonymous: anonymous,
  interface: :api
}
```

To:

```
InfluxDB::Rails.current.tags = {
  beta: User.possibly_nobody.in_beta?,
  anonymous: !User.session,
  interface: :api
}
```

I suggest to do it the same way.

Reply:
i like that @dmarcoux

Reply:
the ```User.session``` does already the check if the current is nobody @coolo

Reply:
I just copy pasted what was done by @coolo, so credits go to him. It's making it easier to understand what I'm referring to.

Reply:
Right. @Ana06 pointed out the one case and I missed the other case